modules/kubernetes-addons/aws-load-balancer-controller/variables.tf

```diff
@@ -26,3 +26,8 @@ variable "addon_context" {
   })
   description = "Input configuration for the addon"
 }
+
+variable "default_ecr_uri" {
```

Comment:
nitpick: this variable name should be `ecr_uri`. The default value should be `""`. Also I don't see the check to see if this is non-empty and using this value instead of the actual default value which you retrieve from the map.